packages/core/src/checkout-buttons/checkout-button-strategy-action-creator.ts

```diff
@@ -20,6 +20,14 @@ import {
 import CheckoutButtonRegistryV2 from "./checkout-button-strategy-registry-v2";
 import { CheckoutButtonMethodType, CheckoutButtonStrategy } from "./strategies";
 
+const methodMap: { [key: string]: CheckoutButtonMethodType | undefined } = {
```

Comment:
We will need to address this soon, since we don't want this as a change once we move to V2

Reply:
We will remove this mapper with common PayPalCommerceButtonStrategy as we made with Braintree strategies separation